### DISCUSSION TITLE
Plane instruments not clickeable after a couple of hours - and textures gone?

### DISCUSSION THREAD

**Discussion by Cronch** [flybywiresim/aircraft#2330] [Help] [answered]
Hi guys, Thanks for everything you a re currently doing for this community.
I have a problem. It happened to me twice using the a320 on the trip from SVLR to SKBO.

After around 2hs of flight (I leave the external view while I'm away from the computer) when I open the cockpit view I cannot use anything on the plane and I have to close it (see screenshots attached).
I'm currently using an AVIANCA livery (not sure if this could create the problem or not).
I'm also using the PACX addon.

I'll try using other liveries on the same flight but wanted to know if someone experienced the same thing.

Regards!

![Screenshot 2020-12-09 213436](https://user-images.githubusercontent.com/751078/101705737-5dd13900-3a66-11eb-9009-94f6a9753736.png)
![Screenshot 2020-12-09 213417](https://user-images.githubusercontent.com/751078/101705739-5f026600-3a66-11eb-9c31-b28e7ee825db.png)
![Screenshot 2020-12-09 213353](https://user-images.githubusercontent.com/751078/101705741-5f9afc80-3a66-11eb-8ba1-daa10a072f9e.png)

**Comment by ghost** [comment, marked as answer]
This is a bug with MSFS, it usually resolves itself after a few minutes.

**Reply by Cronch** [reply]
Ok, thanks for the reply. I'll leave it for some time to check if this fix automatically

**Comment by NathanInnes** [comment]
Strangely enough it appears that you have possibly changed the camera view to the outside and then positioned it inside the cockpit. Are you sure you’re in the right camera setting?

**Reply by Cronch** [reply]
What you are saying makes total sense.
Anyway, I'm just using common camera settings. Just changing between showcase, external and cockpit.
But... I'll take a look at the cameras next time it happens to me to see if I can fix that reseting cameras or something.

Thanks for the reply!

**Comment by theersink** [comment]
Could not find a fix for this. Just had it happen. Tried switching res, graphics ect. Nothing solved it. I had to quit out as the textures would not reload.

**Comment by theersink** [comment]
> Hi guys, Thanks for everything you a re currently doing for this community.
> I have a problem. It happened to me twice using the a320 on the trip from SVLR to SKBO.
> 
> After around 2hs of flight (I leave the external view while I'm away from the computer) when I open the cockpit view I cannot use anything on the plane and I have to close it (see screenshots attached).
> I'm currently using an AVIANCA livery (not sure if this could create the problem or not).
> I'm also using the PACX addon.
> 
> I'll try using other liveries on the same flight but wanted to know if someone experienced the same thing.
> 
> Regards!
> 
> ![Screenshot 2020-12-09 213436](https://user-images.githubusercontent.com/751078/101705737-5dd13900-3a66-11eb-9009-94f6a9753736.png)
> ![Screenshot 2020-12-09 213417](https://user-images.githubusercontent.com/751078/101705739-5f026600-3a66-11eb-9c31-b28e7ee825db.png)
> ![Screenshot 2020-12-09 213353](https://user-images.githubusercontent.com/751078/101705741-5f9afc80-3a66-11eb-8ba1-daa10a072f9e.png)

I think I found a cause and the fix for this. Definitely related to external view but I think it will increase the chances if you are also in accelerated time. Had it happen again, this time I went in changed the resolution scale to 200, applied, textures and everything were back. Changed back to normal res scale and no further issues on that flight.
